Agriculture Svg File added which was missing

PrabeshPP opened the pull request:
### What
- Added agriculture.svg file which was missing and was causing error.

### Screenshot
Don't have screenshot ,but here is error that was thrown. And now this error has been solved.
``` shell
unexpected case: asset not found assets/cache/agriculture.svg (https://static.openfoodfacts.org/images/icons/dist/agriculture.svg)

```
### Fixes bug(s)
- #52 
### Part of 
- https://github.com/openfoodfacts/smooth-app/issues/525
(please be as granular as possible)

PrabeshPP committed:
Agriculture Svg File added

monsieurtanuki reviewed (approved):
Thank you @PrabeshPP!
For the record it's not an error, it's more a warning. The file you've just provided in the assets is the fallback size-optimized svg, displayed while the actual server file is downloaded.
I mean, without this file we still get the correct svg file displayed, only a bit later.

monsieurtanuki commented:
@M123-dev @teolemon Any idea what's blocking there? Possible reasons: @PrabeshPP is not (yet) famous enough; the PR does not include code.

M123-dev commented:
Don't know what causes this, but we had this very often lately. Somehow the workflows don't start. @PrabeshPP pushing another commit most of the times fixes the issue

monsieurtanuki committed:
Update agriculture.svg

I've just added a useless space in order to possibly trigger the build process.
